Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -254,7 +254,7 @@ def bottle_formula(f)
             relocatable = false if keg_contain?("#{prefix}/etc", keg, ignores)
             relocatable = false if keg_contain?("#{prefix}/var", keg, ignores)
           end
-          skip_relocation = relocatable && !keg.require_relocation?
+          skip_relocation = relocatable && keg.respond_to?(:require_relocation?) && !keg.require_relocation?
```

Comment:
Ok, thanks. Instead of `respond_to?` could you add a generic, no-op version of that method? Thanks!